package/MDAnalysis/analysis/pcz.py

```diff
@@ -0,0 +1,481 @@
+    s = [5.0, 3.0]
+    x = p.transform(s, inverse=True)
+
+
+Helper functions
+----------------
+The following are convenience classes and functions primarily for internal use.
+
+.. autoclass:: Fasu
+.. autofunction:: lsqfit
+.. autofunction:: rmsd
+
+Classes, methods, and functions
+-------------------------------
+
+.. autoclass:: Pca
+   :members:
+
+   .. attribute:: natoms
+
+      int, number of atoms selected for PC analysis
+
+   .. attribute:: numframes
+
+      int, number of trajectory frames selected for PC analysis
+
+   .. attribute:: avg
+
+      :class:`numpy.ndarray` containing the coordinates of the average structure
+
+   .. attribute:: nvecs
+
+      int, the number of eigenvectors/eigenvalues calculated
+
+   .. attribute:: captured_variance
+
+      float, the fraction (between 0.0 and 1.0) of the total variance captured 
+      by the *nvecs* eigenvectors
+
+   .. attribute:: evals
+
+      :class:`numpy.ndarray` containing the eigenvalues (largest first)
+
+   .. attribute:: evecs
+
+      :class:`numpy.ndarray` containing the eigenvectors
+
+"""
+import MDAnalysis as mda
+import numpy as np
+
+from MDAnalysis.analysis.align import rotation_matrix
+from scipy.linalg import eigh
+
+
+def lsqfit(mobile, reference, weights = None):
+    """
+    Utility function to fit the coordinates in the [n, 3] numpy array
+    *mobile* to those in the [n, 3] array *reference*, with optional mass
+    weighting.
+
+    :Arguments:
+      *mobile*
+        :class:`numpy.ndarray` containing the coordinates to be fitted
+
+      *reference*
+        :class:`numpy.ndarray` containing the target coordinates
+
+      *weights*
+        :class: numpy.ndarray`  (optional) for mass-weighted fitting
+
+    :Returns:
+       :class:`numpy.ndarray` containing the transformed coordinates of *mobile* 
+    """
+    lmob = mobile - mobile.mean(axis=0)
+    targ_com = reference.mean(axis=0)
+    ltarg = reference - targ_com
+    R, rms = rotation_matrix(ltarg, lmob, weights = weights)
+    fitted = np.dot(lmob, np.array(R))
+    fitted += targ_com
+    return fitted
+
+def rmsd(mobile, reference, weights = None):
+    """
+    Utility function to calculate the rmsd between the coordinates in 
+    the [n, 3] numpy array *mobile* and those in the [n, 3] array
+    *reference*, with optional mass weighting.
+
+    :Arguments:
+      *mobile*
+        :class:`numpy.ndarray` containing the coordinates to be fitted
+
+      *reference*
+        :class:`numpy.ndarray` containing the target coordinates
+
+      *weights*
+        :class: numpy.ndarray`  (optional) for mass-weighted fitting
+
+    :Returns:
+       float, the RMS deviation between *mobile* and *reference* after
+       least-squares fiting (with optional mass-weighting) 
+    """
+    lmob = mobile - mobile.mean(axis=0)
+    targ_com = reference.mean(axis=0)
+    ltarg = reference - targ_com
+    R, rms = rotation_matrix(ltarg, lmob, weights = weights)
+    return rms
+
+class Fasu:
```

Comment:
When you push the changes to your github repository github will automatically mark all discussion as resolved where code has been changed.